### DISCUSSION TITLE
Hyper-optimized Techempower/libreactor implementation on AWS - 1.2M reqs with just 4vCPUs

### DISCUSSION THREAD

**Discussion by talawahtech** [TechEmpower/FrameworkBenchmarks#6597] [General]
I've been working on this post/project for a while now. Didn't really want to spam the forum, but I figured people here would interested in something like this.

https://talawah.io/blog/extreme-http-performance-tuning-one-point-two-million/

**Comment by bhauer** [comment]
This is incredible work, @talawahtech.

Everyone, please feel free to post this sort of thing on the discussion forum.  I thoroughly enjoy hearing about how people are putting this project to use in the world.  The research in Marc's article is a deep reservoir of learning material.  I have to admit most of it goes _whoosh_ right over my head on first and second reading.

**Comment by ghost** [comment]
Can you use monoio or a similar io_uring-based library for testing again？
@talawahtech
